designs/virtualization/4.x/expose-guest-data/expose guest data.md

```diff
@@ -0,0 +1,53 @@
+# Expose Guest Agent Data
+
+When the Guest Agent is installed on a VM some additional valuable data can be collected then surfaced in the UI.
+When the Guest Agent isn’t installed we will inform the users about that.
+
+We can report that in the VM’s Overview tab under the Status card and provide a message: “This virtual machine does not have a Guest Agent installed. Some metrics, and management features will not be available or accurate” and add an external link to view the details about the installation procedure of the Guest Agent, if the users are interested to do so. The link will lead the users to the instructions on how to install the Guest Agent. Linux users will need to download and install the Guest Agent online, while Windows users will get the installation on a CD ROM.
+After the installation they will need to restart the VM.
```

Comment:
Should the text in the status state that they need to restart?

Reply:
+1

Reply:
`Linux users will need to download and install the Guest Agent online, while Windows users will get the installation on a CD ROM.`

This is inaccurate, linux and windows users will need to install the GA using the instructions in the link, we do not know if linus users will need to use the software manager to install the GA or not, and we do not know if windows users will have the a CD with the installer.

`After the installation they will need to restart the VM`

we do not know that either.

My 2c are just tell people to use the instructions in the link ...

Reply:
Thanks @yaacov